<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="shuffle_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>110</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>shuffle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hint_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>150</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>hint</string>
   </property>
  </widget>
  <widget class="QLabel" name="level_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
